lib/graphs.js

```diff
@@ -86,6 +86,9 @@ function listOutputTransform(headers, data) {
   return data;
 }
 
+/**
```

Comment:
If this does the trick, add a comment here explaining why this annotation is here, as a future developer won't know why it's here and may remove it, not knowing what it does.